config/configtelemetry/configtelemetry.go

```diff
@@ -128,3 +153,13 @@ func parseLevel(str string) (Level, error) {
 	}
 	return LevelNone, fmt.Errorf("unknown metrics level %q", str)
 }
+
+// GetMetricsAddrDefault returns the default metrics bind address and port depending on
+// the current build type.
+func GetMetricsAddrDefault() string {
+	return ":8888"
+}
+
+func GetMetricsAddr() string {
+	return *metricsAddrPtr
+}
```

Comment:
Consider to remove the public surface, and add "experimental" to all these APIs. I think we can make them deprecated already since we are intended to remove them correct?